lib/praxis/api_definition.rb

```diff
@@ -97,17 +97,21 @@ def describe
         description( description || 'Standard response for successful HTTP requests.' )
 
         media_type media_type
-        location location
-        headers headers if headers
+        location if location
+        headers&.each do |(name, value)|
+          header(name: name, value: value)
```

Comment:
Making name a keyword arg seems to run counter to how we've done attribute like things everywhere else, where the name would just be the first positional arg.

Reply:
good call, changed